samples/RaspberryPiInstructions.md

```diff
@@ -86,8 +84,8 @@ Pete Gallagher has also created single line install scripts for .NET Core and .N
 
 Please refer to Pete Gallagher's Blog;
 
-- [.NET Core](https://bit.ly/dotnetcorepi)
-- [NET 5](https://bit.ly/dotnet5pi)
+* [.NET Core](https://bit.ly/dotnetcorepi)
```

Comment:
in one file, you replace - with *, and here you did the opposite. what's the preferred one?

Reply:
This is a result of the VS Code, automatically applying the first used char on all lists.

Reply:
Gotcha.